components/infieldbutton/metadata/infieldbutton.yml

```diff
@@ -13,6 +13,10 @@ sections:
 
```

Comment:
while in here could we update the name to be `In-Field button`

Reply:
I have it as `In-field button` because I thought only the first letter was supposed to be capitalized... but this one is a weird one with the hyphen